awsConfig.ts

```diff
@@ -0,0 +1,19 @@
+export const awsConfig = {
+    accessKeyId: "AKIAIOSFODNN7EXAMPLE", // Placeholder, avoid hardcoding AWS credentials
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Remove hardcoded AWS credentials</b></div>
<div id="fix">

Consider removing hardcoded AWS credentials and using environment variables consistently. Line 2 contains a hardcoded `accessKeyId` which is overwritten in line 3, while line 4 has a hardcoded `secretAccessKey`. This poses a security risk.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,6 +1,4 @@
 export const awsConfig = {
 -    accessKeyId: "AKIAIOSFODNN7EXAMPLE",
     accessKeyId: process.env.AWS_ACCESS_KEY_ID || "",
 -    secretAccessKey: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
 +    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY || "",
     region: "us-east-1"
 };
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/24#issuecomment-2514653600>#6ccb3d</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged